firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigGetParameterHandler.java

```diff
@@ -92,14 +102,14 @@ public ConfigGetParameterHandler(
    * @param key A Firebase Remote Config parameter key.
    */
   public String getString(String key) {
-    String activatedString = getStringFromCache(activatedConfigsCache, key);
-    if (activatedString != null) {
-      return activatedString;
+    ConfigAndValue configAndValue = getStringFromCache(activatedConfigsCache, key);
```

Comment:
Thinking: customers can also get params via `getAll` ( https://firebase.google.com/docs/reference/android/com/google/firebase/remoteconfig/FirebaseRemoteConfig#getAll() ), which won't log. It feels relatively straightforward to say that's an unsupported advanced use-case, but it is an inconsistency.

Reply:
getAll() relies on getValue(), which does log, so this case is supported.